test/gtest/interpreter.cpp

```diff
@@ -37,7 +37,9 @@ TEST(InterpreterTest, AddI32) {
   auto expr = builder.build();
   ASSERT_FALSE(expr.getErr());
 
-  auto results = Interpreter{}.run(*expr);
+  Interpreter interpreter;
```

Comment:
Now that we don't need to add an instance to the interpreter in these tests, these changes to put the interpreter in a variable aren't strictly necessary. Feel free to revert them if you want, but if you prefer it this way, that's fine, too.

Reply:
done